pages/dkp/kommander/1.3/workspaces/index.md

```diff
@@ -23,33 +23,11 @@ At the **Workspace** level, devops administrators manage multiple clusters withi
 
 At the **Projects** level, devops administrators or developers manage configuration and services across multiple clusters.
 
-#### Default Workspace
+## Default Workspace
 
-To get started immediately you can use the default workspace deployed in Kommander. Your Workspace delegation can be done at a later time.
+To get started immediately, you can use the default workspace deployed in Kommander. Your workspace delegation can be done at a later time.
 
-#### Creating new Workspaces
+# Workspace Procedures
 
-From the workspace selection dropdown at the top of the screen, select **Create Workspace**. Enter a name and description. The workspace is now accessible from the workspace selection dropdown.
-
-![Organization / Workspace Selection](/dkp/kommander/1.3/img/org-nav.png)
-<br />_Organization / Workspace Selection_
-
-![Create Workspace Form](/dkp/kommander/1.3/img/create-workspace.png)
-<br />_Create Workspace Form_
-
-##### Advanced Options
-
-![Workspace Form Advanced Annotations](/dkp/kommander/1.3/img/workspace-annotations.png)
-<br />_Create Workspace Advanced Options_
-
-When creating, or editing, a workspace you can use the Advanced options to add, edit or delete annotations and labels to your workspace. Both the annotations and labels will be applied to the Workspace namespace.
-
-Labels that are added to a Workspace, will also be applied to all of that Workspace’s clusters.
-
-#### Editing and Deleting Workspaces
-
-A new Kubernetes namespace, with your workspace name, is created in the management cluster. All workspace assets are created in this new namespace. The namespace name cannot be changed but the display name and description can be modified.
-
-<p class="message--note"><strong>NOTE: </strong>
-  Workspaces can only be deleted if all the clusters belonging to it have been deleted.
-</p>
+- [Creating new workspaces](/dkp/kommander/1.3/workspaces/creating-workspaces/)
```

Comment:
It could just be like this - that way we won't have to worry about versioning or anything like that as well, because it's a child of this page.
```suggestion
- [Creating new workspaces](creating-workspaces/)
```